juju/charmhub.py

```diff
@@ -0,0 +1,24 @@
+from .client import client
+from .errors import JujuError
+
+
+class CharmHub:
+    def __init__(self, model):
+        self.model = model
+
+    async def info(self, name, channel=None):
+        """info displays detailed information about a CharmHub charm. The charm
+        can be specified by name or by path.
+
+        """
+        if not name:
+            raise JujuError("name expected")
+
+        if channel is None:
+            channel = "latest"
```

Comment:
A channel cannot be a track only.  It'll fail at some point on the controller.  I'd prefer leaving channel blank if unspecified.  The caller won't know which channels are available.

Reply:
Yes it can, channels are normalised from `latest/stable` to `latest`...